cnf-tests/Dockerfile.konflux

```diff
@@ -0,0 +1,74 @@
+FROM brew.registry.redhat.io/rh-osbs/openshift-ose-cli-rhel9:v4.19 AS oc
```

Comment:
Is there a reason for drifting from `registry.ci.openshift.org` to `brew.registry.redhat.io`?

Reply:
`brew.registry.redhat.io` is the external avialable link for `registry-proxy`, this was intended because we are doing a mapping from midstream to u/s and in m/s `registry-proxy` was used